### PR TITLE
Sidebar nav. items' design fix

### PR DESCRIPTION
**Pull request by icoderharshit**
• Improved font-family, font-size, letter-spacing for the nav .items in the sidebar !
-imported Google Font "Merienda"
• Increased space between nav .items in sidebar and improved transition speed of the underline coming on hover by 0.1s

Issue: Change in Sidebar Nav .items Font Family ! #247
Text in the Nav. items of sidebar looks really congested and font family also looks not so aesthetic.
• Attaching the before after look of the sidebar .nav items below ⬇
bugfix

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
I like what you've done with the overall style, but I think it would be better if we use "Syne" font, which is already imported in `index.html`. I think it fits the style of the website more. Can you change the font used for the sidebar items to that instead?

**Comment by icoderharshit** [comment]
Yes, sure...will be implementing the requested change ! Check it out, by tomorrow morning :)

**Comment by icoderharshit** [comment]
I did the requested change ! Check it out, please @shahednasser 👍 I hope it helps and the website's sidebar is looking better than before with improved readability and smooth transition with more space in between the sidebar .nav items, looking forward to contribute more to this amazing open-source project.
• Really excited to see my first Open-Source contribution's request getting merged ! Thank you @shahednasser.

File: assets/css/index.css [hunk 1]
```diff
@@ -17,7 +16,7 @@ footer {
 }
 
 .sidebar-list a {
-  font-family: 'Merienda', cursive;
+
```
Comment (shahednasser) [inline assets/css/index.css:19]: Instead of removing Merienda, replace it with "Syne" please.

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: assets/css/index.css (hunk 1 above)
Comment (icoderharshit) [reply assets/css/index.css]: Done, please check :)

**Comment by shahednasser** [comment]
Awesome work thank you!

@all-contributors please add @icoderharshit for code and design

**Comment by icoderharshit** [comment]
> Awesome work thank you!
> 
> @all-contributors please add @icoderharshit for code and design

I would love to contribute more to sbuttons, looking forward to work on improving the overall UI of the website, will be updating soon and create an issue for the same.

**Pull request merged**